ovs/cli/misc/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2019 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+
+from .misc import collect_logs
+from ovs_extensions.cli import OVSGroup
+
+misc_group = OVSGroup('miscellaneous', help='Miscellaneous options')
```

Comment:
this is not a Group. right now, one needs to call
```ovs miscellaneous collect logs```
this is not how it used to be